proposals/working-group/Comparisons.adoc

```diff
@@ -0,0 +1,36 @@
+= Pros/Cons of Each Proposal
+
+== MPWG
+Pros
+
+* Flatter governing structure as steering committee, marketing committee and specification committees share the same membership.
+* Addresses IP gap in existing MP development
+* Provides complete autonomy of how MP evolves as governing structure is not part of a non-MP specific working group
+* Provides a lower barrier to entry in terms of both cost and process for the MP technology
+* Continues a model that has produced multiple independent implementations of novel technologies over a number of years
+* Allows for large companies to sponsor specification development at a zero cost entry point
```

Comment:
Explain? What is preventing this in the other proposal?

Reply:
Currently, to participate on a Jakarta EE Spec Project, you either need to be an independent or you need to be an employee of a Working Group member (which costs something).  Just today on the Steering Committee call, we discussed a potential means of lifting that requirement.  So, maybe this point will no longer apply?

Reply:
Correct. This was brought up in the steering committee meeting and the EF said they would consider it, but it has not manifested. This can be dropped when adopted.